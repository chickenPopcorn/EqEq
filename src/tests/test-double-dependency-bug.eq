Foo = {
  twice = b + b;
}

Foo:find twice {
  b = 10;
  print("twice = %0.f [b = %0.f]\n", twice, b);
}
