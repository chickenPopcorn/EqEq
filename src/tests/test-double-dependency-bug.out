twice = 20 [b = 10]
